plugins/cue_points/session_cue_point/lib/model/filters/SessionCuePointFilter.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * @package plugins.sessionCuePoint
+ * @subpackage model.filters
+ */
+class SessionCuePointFilter extends CuePointFilter
+{
+	public function init()
+	{
+		parent::init();
+		
+		$extendedFields = kArray::makeAssociativeDefaultValue(array(
```

Comment:
no need to implement this and override the parent function, all the fields you need exist on CuePointFilter